src/lib/visualizador/visualizador.component.ts

```diff
@@ -0,0 +1,141 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+export interface FileObject {
+    url: String;
+    ext: String;
+}
+
+export type PlexVisualizadorItem = FileObject | string;
+
+@Component({
+    selector: 'plex-visualizador',
+    template: `
+    <div *ngIf="opened" class="lightbox hover" (click)="close()">
+        <i (click)="previous();$event.stopPropagation();" class="mdi mdi-chevron-double-left previous parpadeo"></i>
+
+        <img [src]="imageSrc" alt="" *ngIf="isImage" (click)="imageClick();$event.stopPropagation();">
+        <a [href]="imageSrc" target="_blank" *ngIf="!isImage" class="adjunto m-1 p-2"
+        (click)="imageClick();$event.stopPropagation();">
+            {{ desc }}
+        </a>
+
+        <i (click)="next();$event.stopPropagation();" class="mdi mdi-chevron-double-right next parpadeo"></i>
+    </div>
+    `
```

Comment:
Me gustó la sugerencia de @JSantarelli! se puede agregar esto para que ande: 
`host: {
        '(document:keydown)': 'detectKeyNavigation($event)'
    }`
Y agregar una función simplecita para las teclas :arrow_left:  y :arrow_right: 
`detectKeyNavigation(e: KeyboardEvent) {
        if (e.which === 37) {
            return this.previous();
        }
        if (e.which === 39) {
            return this.next();
        }
    }`